cluster/gce/windows/configure.ps1

```diff
@@ -136,6 +136,8 @@ try {
   Setup-ContainerRuntime
   DownloadAndInstall-AuthPlugin
   DownloadAndInstall-KubernetesBinaries
+  DownloadAndInstall-CSIProxyBinaries
+  Start-CSIProxy
```

Comment:
Why starting the CSI proxy here? Is the ordering intentional? If so, could you please add a comment describing why this is the case?

Reply:
csi-proxy can be started independently from other kubelet services. So the ordering should not matter.Do you think it is better to move it to Start-WorkerServices?

Reply:
Nope. I think it's better that it starts before kubelet. Just making sure there's no other dependency.

Reply:
sure